lua/dap/session.lua

```diff
@@ -413,7 +413,7 @@ local function jump_to_frame(cur_session, frame, preserve_focus_hint)
     return
   end
   if not source.sourceReference or source.sourceReference == 0 then
-    if not source.path then
+    if not source.path or vim.fn.filereadable(source.path) == 0 then
```

Comment:
This would break cases where the path is a custom uri that can be opened using a BufReadCmd

Reply:
Thanks for the review! 

Do you prefer reverting it? I'm not sure how to handle that.

Reply:
Yep please drop the second commit, that will need a different solution.  The first commit looks good 👍

Reply:
Done, thanks!